tt_metal/tools/profiler/op_profiler.hpp

```diff
@@ -356,7 +356,7 @@ inline std::string op_meta_data_serialized_json(
         j["performance_model"]["input_bws"] = perfModel.get_input_bws();
         j["performance_model"]["output_bws"] = perfModel.get_output_bws();
 
-        std::string short_str = fmt::format("`TT_DNN_DEVICE_OP: {}, {}, {}, ", j["op_code"], program_hash, device_id);
+        std::string short_str = fmt::format("`TT_DNN_DEVICE_OP: {}, {}, {}, ", j["op_code"].dump(), program_hash, device_id);
```

Comment:
Couldn't infer it by itself anymore?

Reply:
The previous `fmt::format` did not produce a compilation error, it checked at runtime and failed. The current version produces a compilation error because there is no format for `nlohmann::json` type.